test/deploy/kubernetes/deploy.sh

```diff
@@ -163,16 +201,20 @@ if [ $REDEFPODS = true ]; then
 
     # Create HA Node (POD)
     print_header "Creating HA Node - Cluster";
-    kubectl apply -f "$BASEPATH/provisioner-pods/deployment_ha_node.yaml" --namespace "$NAMESPACE";
+    kubectl apply -f "$BASEPATH/runtime-pods/components_ha_node.yaml" --namespace "$NAMESPACE";
 
     # Wait for HA Node (POD) execution
     printf "\nWaiting for HA Node.";
-    while [ "$(kubectl get pods | grep ha-node | awk '{print $3}')" != "Completed" ]; do
+    while [ "$(kubectl get pods | grep ha-node | awk '{print $3}')" != "Running" ]; do
```

Comment:
**Improvement:**
We should check for status such as `Failed` and `Unknown` as well

Reply:
Will take it up later.